models/README.md

```diff
@@ -0,0 +1,16 @@
+# Models
+
+Please refer [training]() section to understand how our best model is trained
```

Comment:
refer TO

Reply:
I added a URL for now, I will re-verify URLs after merge to master.